srcpkgs/btop/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'btop'
+pkgname=btop
+version=1.0.24
+revision=1
+build_style=gnu-makefile
+make_build_args="ARCH=\${XBPS_TARGET_MACHINE} ADDFLAGS=\${CXXFLAGS}\${LDFLAGS}"
+make_use_env=true
+short_desc="Monitor of resources"
+maintainer="RunningDroid <runningdroid@zoho.com>"
+license="Apache-2.0"
+homepage="https://github.com/aristocratos/btop"
+distfiles="https://github.com/aristocratos/btop/archive/refs/tags/v${version}.tar.gz"
+checksum=e92da56853c105d3fe62a5b175c1bd3b9a9ce69b9b9a274f0c5f2ad21363c7e2
+make_check=no # no tests
```

Comment:
Why is this required? When a package doesn't have tests, they are automatically skipped.

Reply:
I had just added it to silence the warning, but I'll remove it.